code/datums/storage/storage.dm

```diff
@@ -209,6 +213,9 @@
 	RegisterSignal(parent, COMSIG_ATOM_EMP_ACT, PROC_REF(on_emp_act))
 	RegisterSignal(parent, COMSIG_ATOM_CONTENTS_WEIGHT_CLASS_CHANGED, PROC_REF(contents_changed_w_class))
 
+	if(click_alt_open)
+		RegisterSignal(parent, COMSIG_CLICK_ALT, PROC_REF(on_click_alt))
```

Comment:
Mentioned this on discord but I think it should register regardless in this context:

Storage datums do a lot of 
```dm
atom_storage.width = LONG
atom_storage.size = BIG
atom_storage.open_on_click = TRUE
atom_storage.silent = TRUE
```
And if someone tried to use that pattern on this var, it would not work.